Signal/src/call/OWSWebRTCDataProtos.pb.m

```diff
@@ -107,13 +107,13 @@ - (OWSWebRTCProtosConnectedBuilder*) toBuilder {
 }
 - (void) writeDescriptionTo:(NSMutableString*) output withIndent:(NSString*) indent {
   if (self.hasId) {
-    [output appendFormat:@"%@%@: %@\n", indent, @"id", [NSNumber numberWithLongLong:self.id]];
+    [output appendFormat:@"%@%@: %@\n", indent, @"id", [NSNumber numberWithLongLong:(long long)self.id]];
```

Comment:
This is generated code.

Reply:
Ugh. Didn't realize. Now I notice the `.pb.m` 🤦‍♂️ 

Will revert.